src/wp-admin/css/about.css

```
@@ -48,14 +46,14 @@
 .credits-php,
 .freedoms-php,
 .privacy-php {
-	background: #f0f7ff;
+	background: var(--about-page--background);
 }
 
 .about-php #wpcontent,
 .credits-php #wpcontent,
 .freedoms-php #wpcontent,
 .privacy-php #wpcontent {
-	background: linear-gradient(180deg, #fff 50%, #f0f7ff 100%);
+	background: linear-gradient(180deg, var(--about-page--background) 50%, var(--wp-admin--surface--background) 100%);
```

I think this is reversed? Should it be 
```suggestion
	background: linear-gradient(180deg, var(--wp-admin--surface--background) 50%, var(--about-page--background) 100%);
```